openmicroscopy/openmicroscopy
# Spw grid view2

# Spatial view of fields in Wells

2nd PR working towards https://github.com/openmicroscopy/design/issues/52
Builds on https://github.com/openmicroscopy/openmicroscopy/pull/4863

Also see https://trello.com/c/GxqDnD0P/211-spw-follow-up

To test:
 - Browse SPW, click on Well. Well 'bird's eye' spatial view should open below tree.
 - Selecting images in central lower panel should highlight these in bird's eye view.
 - Selecting images in bird's eye by clicking / shift-clicking or dragging should highlight same images in central lower panel and right panel.
 - Bird's eye panel will stay open until X in top right corner is clicked.
 - Will be shown again when Well is clicked.
 - Enter a value in the "Wrap [ ] images per row" field. On 'Enter' this should wrap the images in each well below. Should also wrap any new wells loaded.
 - Remove wrapping by clearing the input (or enter 0).

TODO:
- [x] integration tests for posX and posY 

![screen shot 2016-11-14 at 16 46 02](https://cloud.githubusercontent.com/assets/900055/20295500/361a85f8-aafd-11e6-9d4a-8c5f219c19ed.png)


![screen shot 2016-11-14 at 16 19 49](https://cloud.githubusercontent.com/assets/900055/20272589/48fcb96a-aa86-11e6-8633-f1db0ae178fe.png)
